src/lib/channels.py

```diff
@@ -52,6 +56,9 @@ class Channels(object):
     ]
 
     RADIO_MAGAZINE_RANGES = [
```

Comment:
Can we consistent and call this MAGAZINE everywhere?

Reply:
Will keep datasets and scheme names as s01mag0X since this were pushed to firebase ealier on on Monday  and Flow names as mag_s01e0x_activation

Reply:
Analysis file header names will be named fully as MAGAZINE and DRAMA to enable external RDA understand the difference between the two.